ERCS/erc-draft-wallet-connection-api.md

```diff
@@ -0,0 +1,180 @@
+---  
```

Comment:
```suggestion
---  
eip: 7846
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.